KEYS

```diff
@@ -0,0 +1,148 @@
+This file contains the PGP keys of developers who have signed releases of Helm.
+
+For convenance commands are provided for those who use pgp and gpg.
+
+For users to import keys:
+       pgp < KEYS
+       gpg --import KEYS
+Developers to add their keys: 
+        pgp -kxa <your name> and append it to this file.
+        (pgpk -ll <your name> && pgpk -xa <your name>) >> this file.
+        (gpg --list-sigs <your name>
+             && gpg --armor --export <your name>) >> this file.
+
+pub   rsa4096/0x461449C25E36B98E 2017-11-10 [SC]
```

Comment:
Should a separator go above this to clearly mark the commands from the keys info?